src/routes/login/+page.svelte

```diff
@@ -1,15 +1,23 @@
 <script lang="ts">
+	import { run } from 'svelte/legacy';
+
 	import type { PageData } from './$types';
 	import { fade } from 'svelte/transition';
 	import Icon from 'svelte-awesome';
 
 	import { faDiscord } from '@fortawesome/free-brands-svg-icons/faDiscord';
 
-	export let data: PageData;
-
-	$: if (data.redirectAfterLoginURL) {
-		sessionStorage.setItem('redirectAfterLoginURL', data.redirectAfterLoginURL);
+	interface Props {
+		data: PageData;
 	}
+
+	let { data }: Props = $props();
+
+	run(() => {
```

Comment:
same here?

Reply:
This is the one case of the four above we'd use `$effect`:
```js
$effect(() => {
	if (data.redirectAfterLoginURL) {
		sessionStorage.setItem('redirectAfterLoginURL', data.redirectAfterLoginURL);
	}
});
```